api/cpp/include/slint.h

```diff
@@ -1261,13 +1261,29 @@ struct [[deprecated]] VersionCheckHelper
 };
 }
 
+/// Enum for the event loop mode parameter of the slint::run_event_loop() function.
+/// It is used to determine when the event loop quits
+enum class EventLoopMode {
+    /// The event loop will quit when the last window is closed
+    /// or when slint::quit_event_loop() is called.
+    QuitOnLastWindowClose,
+
+    /// The event loop will keep running until slint::quit_event_loop() is called,
+    /// even when all windows are closed.
+    KeepAlive
```

Comment:
I suggest to name it like in Rust.

```suggestion
    RunUntilQuit
```